### ISSUE TITLE
ES6 IntelliSense - require ('./myFile') not working

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/nodejstools#1049]
##### Repro steps:
1. Create a new “Starter Azure Node.js Express 3” project.
2. Create “hello” folder under project root folder.
3. Create “item1.js” under “hello”
4. Type follow code in “item1.js”:
```js
module.exports.a = 3;
module.exports.b = "hi"; 
```
5.	Back in app.js, add the following at the end of the file:
```js
        var item = require('./hello/item1');
        item.
```
##### Expected: 
1. List of completions should be displayed after typing the . in item., starting with a and b.
2. completion list items a and b have a (non-warning) glyph to the left-hand side of them.
 
##### Actual
didn't displayed after typing the . in item., starting with a and b.
![image](https://cloud.githubusercontent.com/assets/762848/16054970/3e7dd544-3224-11e6-9e58-9d7e3c186eec.png)

**Comment by mousetraps** [comment]
This may be a dupe of #817, but the issue is occurring with latest U3 bits.

**Comment by mjbvz** [comment]
Just to clarify, this happened on Update 3 RC with the Typescript prerelease installed?

**Comment by mousetraps** [comment]
No, this happened on one of our latest internal builds - 25410.00, which should have the latest TS insertions. @paulvanbrenk is looking into it.

**Comment by paulvanbrenk** [comment]
I do repo with your sample, the following work for me though:
```javascript
var Animals = {
    Mammals: {
        CAT: 1,
        RAT: 2
    },
    Birds: {
        CROW: 3
    }

}

module.exports = Animals;
```

**Comment by paulvanbrenk** [comment]
So your specific example will be fixed in TS 2.0 https://github.com/Microsoft/TypeScript/issues/7824

**Comment by paulvanbrenk** [comment]
This will be fixed in VS2015 Update 3
